catalog-info.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: backstage.io/v1alpha1
```

Comment:
```suggestion
# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
apiVersion: backstage.io/v1alpha1
```

Could you please add this `yaml-language-server` line in the Buildkite resource manifest? This is very helpful for live validation/completion for RREs when using, for instance, Visual Code 
Thank you very much :)!